test/test_mkldnn_fusion.py

```diff
@@ -0,0 +1,110 @@
+    elif hasattr(F, name):
+        return getattr(F, name)
+    else:
+        raise NameError("Eltwise function %s not found" % name)
+
+
+@unittest.skipIf(not torch._C.has_mkldnn, "MKL-DNN build is disabled")
+class TestMkldnnFusion(JitTestCase):
+    def assertFused(self, graph, fused_patterns):
+        for pat in fused_patterns:
+            self.assertGraphContainsExactly(graph, pat, 0)
+
+    def _check_model(self, m, x):
+        old_fusion_inlining = torch._C._debug_get_fusion_group_inlining()
+        torch._C._debug_set_fusion_group_inlining(False)
+
+        old_cpu_fuser_state = torch._C._jit_can_fuse_on_cpu()
+        torch._C._jit_override_can_fuse_on_cpu(True)
+
+        old_te_must_use_llvm_cpu = torch._C._jit_get_te_must_use_llvm_cpu()
+        torch._C._jit_set_te_must_use_llvm_cpu(False)
+
+        m.eval()
+        with torch.no_grad():
+            script = torch.jit.script(m)
+        script = torch.jit.freeze(script)
+
+        with torch.no_grad():
+            y = warmup_and_run_forward(script, x)
+            y = script(x)
+            y_ref = m(x)
+
+            graph = script.graph_for(*x)
+            self.assertEqual(y, y_ref)
+
+        torch._C._debug_set_fusion_group_inlining(old_fusion_inlining)
+        torch._C._jit_override_can_fuse_on_cpu(old_cpu_fuser_state)
+        torch._C._jit_set_te_must_use_llvm_cpu(old_te_must_use_llvm_cpu)
+        return graph
+
+    def test_conv(self):
+        class M(nn.Module):
+            def __init__(self, in_channels, out_channels, bias, **kwargs):
+                super(M, self).__init__()
+                self.conv = torch.nn.Conv2d(in_channels, out_channels, bias=bias, **kwargs)
+
+            def forward(self, x):
+                res = self.conv(x)
+                return res
+
+        for memory_format, enabled in [
+            [torch.contiguous_format, True],
+            [torch.channels_last, False],  # TODO: enable support on channels_last
+        ]:
+            for bias in [True, False]:
+                m = M(3, 10, bias, kernel_size=(3, 3)).to(memory_format=memory_format)
+                x = torch.randn(1, 3, 224, 224).to(memory_format=memory_format)
+                graph = self._check_model(m, x)
+                if enabled:
+                    self.assertFused(graph, ['aten::conv2d'])
+                    self.assertGraphContainsExactly(graph, FUSION_GROUP, 1)
+                else:
+                    self.assertGraphContains(graph, kind='aten::conv2d')
+
+    def test_conv_eltwise(self):
+        class M(nn.Module):
+            def __init__(self, eltwise_fn, in_channels, out_channels, bias, **kwargs):
+                super(M, self).__init__()
+                self.conv = torch.nn.Conv2d(in_channels, out_channels, bias=bias, **kwargs)
+                self.eltwise = eltwise_fn
+
+            def forward(self, x):
+                x = self.conv(x)
+                x = self.eltwise(x)
+                return x
+
+        for eltwise in ["relu"]:
```

Comment:
Would it be simpler if we had a list of functions themselves instead of a name by which we need to look them up?
I.e.:
```
for eltwise_fn in [torch.nn.functional.relu, ...]:
  ...
```

Reply:
Thanks for the suggestion! I've modified it to directly iterate through functions instead of looking them up (https://github.com/pytorch/pytorch/pull/77157/commits/aeaf507cf964559ad0003a5ea5ed0a6c44b2d50c).